### ISSUE TITLE
Fix issue with jit-fusion on test-miniz and enable in rv-jit

### ISSUE THREAD

**Issue by michaeljclark** [rv8-io/rv8#41]
jit-fusion.h contains a simple state machine that matchs CALL (AUIPC+JALR) and LA (AUIPC+ADDI) to coalesce them into one single operation. It is intended that the state machine can coalesce other instruction sequences however there is currently a bug in the fusion state machine that appears on test-miniz so it is currently disabled.

```
git diff 452537c51871d6f5670d22b32153e6f294411b6f..edd60409afead0f95628985c98fcb58e0811c000
--- a/src/app/rv-jit.cc
+++ b/src/app/rv-jit.cc
@@ -91,13 +91,13 @@ using proxy_model_rv32imafdc = processor_rv32imafdc_model<
        jit_decode, processor_rv32imafd, mmu_proxy_rv32>;
 using proxy_jit_rv32imafdc = jit_runloop<
        processor_proxy<proxy_model_rv32imafdc>,
-       jit_fusion<jit_emitter_rv32<proxy_model_rv32imafdc>>>;
+       jit_emitter_rv32<proxy_model_rv32imafdc>>;
 
 using proxy_model_rv64imafdc = processor_rv64imafdc_model<
        jit_decode, processor_rv64imafd, mmu_proxy_rv64>;
 using proxy_jit_rv64imafdc = jit_runloop<
        processor_proxy<proxy_model_rv64imafdc>,
-       jit_fusion<jit_emitter_rv64<proxy_model_rv64imafdc>>>;
+       jit_emitter_rv64<proxy_model_rv64imafdc>>;
 
 /* environment variables */
 ```

**Comment by michaeljclark** [comment]
The JIT fusion engine was fixed in commit 382154b28abdb198de4be5a21a52ba21e982977b however it is still disabled by default in `src/app/rv-jit.cc` until further testing is performed.

**Comment by michaeljclark** [comment]
JIT fusion has been enabled by default
